blurb/blurb.py

```diff
@@ -745,19 +745,18 @@ def test(*args):
 
 
 def find_editor():
-    for var in 'GIT_EDITOR', 'EDITOR':
-        editor = os.environ.get(var)
-        if editor is not None:
-            return editor
```

Comment:
Actually, wait a minute.  Does "git var GIT_EDITOR" still use vi as a default value?  Because I won't allow blurb to run "vi" unless the user has explicitly configured something to use it.